tests/tpch/conftest.py

```diff
@@ -119,38 +118,18 @@ def name(request, tmp_path_factory, worker_id):
     return data
 
 
-@pytest.fixture(scope="function")
-def benchmark_time(test_run_benchmark, module, scale, name):
-    """Benchmark the wall clock time of executing some code.
-
-    Yields
-    ------
-    Context manager that records the wall clock time duration of executing
-    the ``with`` statement if run as part of a benchmark, or does nothing otherwise.
-
-    Example
-    -------
-    .. code-block:: python
-
-        def test_something(benchmark_time):
-            with benchmark_time:
-                do_something()
-    """
-
-    @contextlib.contextmanager
-    def _benchmark_time():
-        if not test_run_benchmark:
-            yield
-        else:
-            start = time.time()
-            yield
-            end = time.time()
-            test_run_benchmark.duration = end - start
-            test_run_benchmark.start = datetime.datetime.utcfromtimestamp(start)
-            test_run_benchmark.end = datetime.datetime.utcfromtimestamp(end)
-            test_run_benchmark.cluster_name = f"tpch-{module}-{scale}-{name}"
```

Comment:
Evidence that we didn't lose cluster_name and we gained cluster_id and cluster_details_url:

session 6b2 is latest nightly run in main
session 84c is this PR
![image](https://github.com/coiled/benchmarks/assets/6213168/7dea6e59-f642-4a11-b16e-99335a01cf3b)